docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningEarlyStopping.cs

```diff
@@ -19,100 +19,135 @@ public class ResnetV2101TransferLearningEarlyStopping
+                    // patience sets the number of epochs to wait to see if there is 
+                    //      any improvement before stopping.
+                    // metric: the metric to monitor
                     EarlyStoppingCriteria = new ImageClassificationTrainer.EarlyStopping(
-                        minDelta: 0.001f, patience: 20, metric: ImageClassificationTrainer.EarlyStoppingMetric.Loss),
+                        minDelta: 0.001f, patience: 20,
+                        metric: ImageClassificationTrainer.EarlyStoppingMetric.Loss),
                     MetricsCallback = (metrics) => Console.WriteLine(metrics),
-                    ValidationSet = validationSet
+                    ValidationSet = testDataset
                 };
 
-                var pipeline = mlContext.Transforms.LoadImages("Image", fullImagesetFolderPath, false, "ImagePath") // false indicates we want the image as a VBuffer<byte>
-                    .Append(mlContext.MulticlassClassification.Trainers.ImageClassification(options));
+                // Create the ImageClassification pipeline
+                var pipeline = mlContext.Transforms.LoadImages("Image", 
+                    fullImagesetFolderPath, false, "ImagePath") 
+                    .Append(mlContext.MulticlassClassification.Trainers.
+                        ImageClassification(options));
 
-                Console.WriteLine("*** Training the image classification model with " +
-                    "DNN Transfer Learning on top of the selected pre-trained " +
-                    "model/architecture ***");
+                Console.WriteLine("*** Training the image classification model " +
+                    "with DNN Transfer Learning on top of the selected " +
+                    "pre-trained model/architecture ***");
 
                 // Measuring training time
                 var watch = System.Diagnostics.Stopwatch.StartNew();
 
+                // Train the model
+                // This involves calculating the bottleneck values, and then
+                // training the final layer. Sample output is: 
+                // Phase: Bottleneck Computation, Dataset used: Train, Image Index: 1
+                // Phase: Bottleneck Computation, Dataset used: Train, Image Index: 2
+                // ...
+                // Phase: Training, Dataset used:  Train, Batch Processed Count: 18,
+                //    Learning Rate:       0.01 Epoch:   0, Accuracy:  0.9166667, 
+                //    Cross-Entropy:  0.4866541
+                // ...
+                // Phase: Training, Dataset used:  Train, Batch Processed Count: 18, 
+                //      Learning Rate:       0.01 Epoch:  19, Accuracy:          1, 
+                //      Cross-Entropy: 0.03978536
+                // Phase: Training, Dataset used: Validation, Batch Processed Count: 3,
+                //      Epoch: 19, Accuracy: 0.852381
+                // We see that the training stops when the metric stops improving.
```

Comment:
Thanks, @aibhanda@microsoft.com ! In this case where limiting the output to 85 characters actually makes it difficult to read sample output it is ok to cross 85 character. Otherwise your change looks very good, please fix similar stuff else where.